sim/src/trips.rs

```diff
@@ -538,26 +538,11 @@ impl TripManager {
             ParkingSpot::Offstreet(b, _) => {
                 self.events
                     .push(Event::PersonEntersBuilding(trip.person, b));
-                // Actually, to unpark, the car's front should be where it'll wind up at the end.
```

Comment:
This correction now happens in `start_car_on_lane`. Getting the math right here is too hard; sometimes we have to subtract or add, depending if we're going to use the offside or not.